palette/Cargo.toml

```diff
@@ -67,10 +68,14 @@ version = "1"
 optional = true
 
 [dependencies.wide]
-version = "0.7.3"
+version = "0.7"
```

Comment:
On the contrary, it's better to be explicit about the lowest supported version. Did it cause any issues?

Reply:
Ahh yes, sorry I tested my implementation in another project by pointing cargo to my fork, and encountered some issues because ultimately two different versions of the `simba` crate got installed in my project, and the palette trait weren't implemented on the versions of the types I used in the application.

Was totally unrelated to this version here as it turned out, but while figuring out what was the issue I thought I quickly change it here to see if it makes any difference.

Have reverted this change